app/Jobs/CreateRockTheVotePostInRogue.php

```diff
@@ -211,64 +222,61 @@ private function parseReferralCode($referralCode)
     }
 
     /**
-     * For a given record and referral code values, first check if we have a northstar ID, then grab the user using that.
-     * Otherwise, see if we can find the user with the given email (if it exists), if not check if we can find them with the given phone number (if it exists).
-     * If all else fails, create the user .
+     * Fetch user ID by given email or mobile if exists. If not found, create new user.
      *
-     * @TODO - If we have a northstar id in the referral code, then we probably don't need to make a call to northstar for the full user object.
```

Comment:
While revisiting the logic for when to use `isset` and `empty` in checking for variables, I thought it made sense to fix this TODO by moving the check for a `northstar_id` column into `handle` instead of passing a `values` parameter here.